Lib/multiprocessing/shared_memory.py

```diff
@@ -76,6 +77,8 @@ def __init__(self, name=None, create=False, size=0):
             raise ValueError("'size' must be a positive integer")
         if create:
             self._flags = _O_CREX | os.O_RDWR
+            if size > _MAX_SHM_SIZE_LIMIT:
+                raise ValueError("'size' cannot exceed " + str(_MAX_SHM_SIZE_LIMIT))
```

Comment:
Please limit lines to 79 characters to follow PEP 8.

Also, I think an f-string should be used here.

Reply:
Fixed